build/build.proj

```diff
@@ -12,6 +12,21 @@
     </CommonMSBuildProperties>
   </PropertyGroup>
 
+  <!--
+    ============================================================
+    Get XPLAT projects
+    ============================================================
+  -->
+  <Target Name="GetXPLATProjects">
```

Comment:
These targets are for source build. 

I am not sure whether they are relevant for your case.